### PR TITLE
Frame validity checks

### PR DESCRIPTION
**Pull request by sean-roberts**
adding height and width heuristic checks to the iframe validity check

### CODE REVIEW

**Comment by robertknight** [comment]
Needs rebasing on master and the base branch changed, assuming you want this to be merged directly into master after review @sean-roberts ?

**Comment by sean-roberts** [comment]
@robertknight rebased

File: src/annotator/util/frame-util.js [hunk 1]
```diff
@@ -37,10 +37,27 @@ function isAccessible (iframe) {
...
+
+/**
+ * Check if the frame elements being considered for injection have the
+ * basic heuristics for content that a user might want to annotate.
+ *  Rules:
+ *    - avoid our client iframe
+ *    - iframe should be sizeable - to avoid the small advertisement and social plugins
+ *
+ * @param  {HTMLIFrameElement} iframe the frame being checked
+ * @returns {boolean}   retult of our validity checks
```
Comment (robertknight) [inline src/annotator/util/frame-util.js]: Typo - "retult"

**Comment by sean-roberts** [comment]
@robertknight addressed review feedback

**Pull request merged**